.github/workflows/env.sh

```diff
@@ -22,14 +22,15 @@ EOF
 
 DEPS_CMAKE_FLAGS=
 FUNCTIONALTEST=functionaltest
-BUILD_FLAGS="CMAKE_FLAGS=-DCI_BUILD=ON -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX:PATH=$HOME/nvim-install -DBUSTED_OUTPUT_TYPE=nvim -DDEPS_PREFIX=$HOME/nvim-deps/usr -DMIN_LOG_LEVEL=3"
+BUILD_FLAGS="CMAKE_FLAGS=-DCI_BUILD=ON -DCMAKE_INSTALL_PREFIX:PATH=$HOME/nvim-install -DBUSTED_OUTPUT_TYPE=nvim -DDEPS_PREFIX=$HOME/nvim-deps/usr -DMIN_LOG_LEVEL=3"
 
 case "$FLAVOR" in
   asan)
     cat <<EOF >> "$GITHUB_ENV"
 CLANG_SANITIZER=ASAN_UBSAN
 ASAN_OPTIONS=detect_leaks=1:check_initialization_order=1:log_path=$GITHUB_WORKSPACE/build/log/asan:intercept_tls_get_addr=0
 UBSAN_OPTIONS=print_stacktrace=1 log_path=$GITHUB_WORKSPACE/build/log/ubsan
+BUILD_FLAGS="$BUILD_FLAGS -D CMAKE_BUILD_TYPE=RelWithDebInfo"
```

Comment:
This doesn't seem to be taking effect -- https://github.com/neovim/neovim/actions/runs/4073768577/jobs/7018093223#step:10:28

Reply:
Thank you. Fixed.